runtime/browser/android/renderer_host/xwalk_render_view_host_ext.cc

```diff
@@ -105,16 +104,6 @@ void XWalkRenderViewHostExt::RenderProcessGone(base::TerminationStatus status) {
   }
 }
 
-void XWalkRenderViewHostExt::DidNavigateAnyFrame(
```

Comment:
because both (Runtime and XWalkRenderViewHost) are WebContentsObserver and they seem to observe the same web content object. xwalk_render_view_host_ext seems to be the added bit to handle Android specific code related to stuff happening with the web content object. I do not see why I should duplicate the same functionality/code. Runtime is used and created in Android so both are going to get the same method calls. That's my understanding.